lib/v2/screens/explore_screens/vote_screens/proposals/proposals_list.dart

```diff
@@ -56,7 +59,21 @@ class _ProposalsListState extends State<ProposalsList> with AutomaticKeepAliveCl
     super.build(context);
     return BlocProvider(
       create: (_) => _proposalsBloc,
-      child: BlocBuilder<ProposalsListBloc, ProposalsListState>(
+      child: BlocConsumer<ProposalsListBloc, ProposalsListState>(
+        listenWhen: (_, current) => current.pageCommand != null,
+        listener: (context, state) async {
+          var pageCommand = state.pageCommand;
+          _proposalsBloc.add(const ClearProposalsListPageCommand());
+          if (pageCommand is NavigateToRouteWithArguments<ProposalsAndIndex>) {
+            int? index = await NavigationService.of(context).navigateTo(pageCommand.route, pageCommand.arguments);
+            if (index != null) {
+              // 420 is the height of card
+              // ignore: unawaited_futures
+              _scrollController.animateTo(420.0 * index,
```

Comment:
can we save this number into a constant and use in both places? 
That way if we need to change we dont forget to change the other place too. 

It can be a constants file for this screen.

Reply:
Currently, this is the only place that has cards with a height of 420

Reply:
Also, I could put it as constant but the fact is that the layout of this card could change in the future and the height is a function of it.

If I remove the comment the developer will not know why the constant does not work.